notebooks/first_steps.ipynb

```diff
@@ -41,9 +41,30 @@
      "start_time": "2018-02-09T18:45:43.533004Z"
     }
    },
-   "outputs": [],
+   "outputs": [
+    {
+     "name": "stderr",
+     "output_type": "stream",
+     "text": [
+      "/u/system/SLES12/soft/pyiron/dev/anaconda3/lib/python3.10/site-packages/pandas/core/computation/expressions.py:21: UserWarning: Pandas requires version '2.8.0' or newer of 'numexpr' (version '2.7.3' currently installed).\n",
+      "  from pandas.core.computation.check import NUMEXPR_INSTALLED\n"
+     ]
+    },
+    {
+     "data": {
+      "application/vnd.jupyter.widget-view+json": {
+       "model_id": "f8dc407474b04f9199b0fa2869ce02a2",
+       "version_major": 2,
+       "version_minor": 0
+      },
+      "text/plain": []
+     },
+     "metadata": {},
+     "output_type": "display_data"
+    }
+   ],
    "source": [
-    "from pyiron import Project"
+    "from pyiron_atomistics import Project"
```

Comment:
Since we are testing this from the `pyiron` repo, I would stick to `from pyiron import Project`

Reply:
Hmmm I see your point, but `pyiron` takes so much more time that it might be a bit discouraging to see this in the very first line

Reply:
Yes, but this is the only test-space to check that the `from pyiron import` is working correctly! Maybe we have to make the lookup faster if it is just too annoying? Or is it because you also have all the other pyiron modules installed which are not loaded in a nice way (especially contrib...). I mean if only
```
conda create --name pyiron pyiron
```
was used it should essentially only find and load `pyiton_atomistics` at a moderate overhead?

Reply:
No, I'm with Niklas here -- this is basically the only place that we might reasonably `from pyiron import...` and then expect that to be providing us with sub module support (atomistics, continuum, etc). So if that's something we want to provide, IMO this is the right place to test it. The fact it's painfully slow is related, but a separate issue.

Reply:
> The fact it's painfully slow is related, but a separate issue.

Well, as long as we cannot solve this problem, it doesn't change the fact that this is the very first line of the very first notebook that we present on our website, and it is extremely demoralising and unsettling to see a notebook frozen for a significant amount of time in the first cell.

Anyway shouldn't we anyway have integration tests if we want to test `from pyiron import Project`?

Reply:
The notebooks run on the CI, so I feel they _are_ the integration tests (albeit by another name and also wearing at least one other hat.)

If importing from pyiron is really such a pain that we don't want people to do it, and we can't fix it, then perhaps we should stop supporting that syntax?

Reply:
They run on the CI, but they are at the same time the example notebooks of the website. For me the CI in this case is there only to guarantee that the example notebooks run, but not really to test pyiron.

I don't think we should stop using `from pyiron import Project`. Instead, I think it's a rather special case where a user wants to use tools across different length scales. After all, we introduced `pyiron_atomistics` to make it possible to import only the modules for the users who are only interested in atomistic tools, and since the notebook is all about atomistic calculations, I think it's correct to present only `pyiron_atomistic` at this point.

Reply:
This being said, I don't have a very strong feeling for this, so I can also put it back if you feel strongly that it should be there.

Reply:
Those are fair points. I still come down on the side of leaving it as from pyiron, but you've sowed enough doubt that I also don't feel strongly. @niklassiemer I'll leave the responsibility of tie breaking to you, but it doesn't sound like either Sam or I will write you out of the will for your choice 😂

Reply:
I am in favor of leaving it `from pyiron` here. However, it might be reasonable to use the notebooks from the `pyiron_atomistics` repo for the education on the website.
Furthermore, we probably should raise an issue for the slow import. 
Some solution could be:
- not import but only find other `pyiron_modules`  than pyiron_atomistics on import of pyiron and import those upon access.
- provide a module scoped list (dict? ) which is used to omit the finding in the first place, i.e. a list like `['contrib', 'gui', ... ]` which is used for autocompletion and to extend this to the corresponding pyiron_module. If a not listed access pyiron.some_mod is performed try to `import pyiron_some_mod` and use this.
- stop treating pyiron as drop in replacement of pyiron_atomistics and actually use pyiron.atomistics (probably together with one of the first). Breaks backwards compatibility, though!

Reply:
> I am in favor of leaving it `from pyiron` here. However, it might be reasonable to use the notebooks from the `pyiron_atomistics` repo for the education on the website.

ok that means I should copy this notebook to `pyiron_atomistics` and create an integration tests folder where we'd have essentially the same content with `from pyiron import Project`?

Reply:
I did not check the exact content, but aren't the notebooks here exactly such copies from pyiron_atomistics with the `from pyiron` to `from pyiron_atomistics` change? It is a not so nice setup, since near duplication, i.e. please apply the same changes over in pyiron_atomistics 😇

Reply:
And I really meant for the purpose of displaying on the webpage. Right now, the tutorial is from this repo, while you claim it to be much faster if we use the notebooks from pyiron_atomistics for the webpage.

Reply:
> I did not check the exact content, but aren't the notebooks here exactly such copies from pyiron_atomistics with the `from pyiron` to `from pyiron_atomistics` change?

AHA!! I didn't know that it existed TWICE!

> And I really meant for the purpose of displaying on the webpage.

I'm not really sure if I understand what you are referring to. Do I understand it correctly:

- We keep `from pyiron import Project` in this repo
- On the website we use the version from `pyiron_atomistics`, i.e. the same notebook with `from pyiron_atomistics import Project`?